Sources/Plasma/PubUtilLib/plResMgr/plRegistryKeyList.cpp

```diff
@@ -201,6 +201,18 @@ bool plRegistryKeyList::SetKeyUnused(plKeyImp* key, LoadStatus& loadStatusChange
     return foundKey != nullptr;
 }
 
+void plRegistryKeyList::PrepForWrite()
+{
+    uint32_t objectID = 1;
+    for (auto it = fKeys.begin(); it != fKeys.end(); ++it) {
+        plKeyImp* key = *it;
```

Comment:
```suggestion
    for (plKey& key : fKeys) {
```

Reply:
This would have to be `for (plKeyImp* key : fKeys) {` instead (since we can't call `SetObjectID` on `plKeyData`), but I'm happy to make that change

Reply:
Whoops! fixed